packages/babel-parser/test/fixtures/experimental/explicit-resource-management/invalid-using-binding-await-module/output.json

```diff
@@ -0,0 +1,166 @@
+{
+  "type": "File",
+  "start":0,"end":113,"loc":{"start":{"line":1,"column":0,"index":0},"end":{"line":12,"column":1,"index":113}},
+  "errors": [
+    "SyntaxError: Can not use 'await' as identifier inside an async function. (2:8)",
```

Comment:
Unrelated, but this error message is wrong (we are not in an async function, but at the top-level of a module).

Reply:
We don't differentiate between module top level and async function as long as `[Await]` production parameter set. Maybe rephrase it as "Can not use 'await' as identifier inside an async context"?